crypto/cryptoerror.go

```diff
@@ -18,13 +18,16 @@ package crypto
 
 import "errors"
 
-const errorinvalidversion = "Invalid version"
-const errorinvalidaddress = "Invalid address"
-const errorinvalidthreshold = "Invalid threshold"
-const errorinvalidnumberofsignature = "Invalid number of signatures"
-const errorkeynotexist = "Key does not exist"
-const errorsubsigverification = "Verification failure: subsignature"
-const errorkeysnotmatch = "Public key lists do not match"
-const errorinvalidduplicates = "Invalid duplicates"
+var (
+	errorinvalidversion           = errors.New("Invalid version")
```

Comment:
errors in go are usually named errXXXX or ErrXXX, depending if they are exported or not. But in either case, the variable naming convention is either camel case or pascal case.